File: 0x-bonus/yegge/largeNum/largeNum.js
const arr = [1,2,3,10,8,100];

const largest = (integers) => {
  let max = -Infinity;

  integers.forEach ((n) => {
    if (n > max) {
      max = n;
    }
  });

  return max
}
console.log(largest(arr));
//console.log(Math.max(...arr));

// reduce function

const largest = (integers) => {
  return integers.reduce( (max, n) => {
    if (n > max) {
      return n;
    } else {
      return max;
    }
  })
};

File: 0x-bonus/yegge/largeNum/large_num.rb
a =  Array.new(1000) { rand(1..100) }


# Find the largest int value in an array

# Straightforward with interation
def largest(integers)
  nax = integers.first

  integers.each do |n|
    max = n if n > max
  end

  max
end

puts a.max
puts largest(a)
